pkg/autoscaler/config/dynamic_config_test.go

```diff
@@ -30,11 +30,13 @@ var configMap = map[string]string{"max-scale-up-rate": "1",
 	"container-concurrency-target-default":    "10.0",
 	"stable-window":                           "4s",
 	"panic-window":                            "5s",
+	"window-panic-percentage":                 "10.0",
```

Comment:
I think this is supposed to be `panic-window-percentage`. Maybe we should also test the resulting dynamic config properties once to avoid this?

Reply:
Whoops!  Fixed.